app/scripts/include/globe.js

```diff
@@ -1,4 +1,3 @@
-'use strict'
```

Comment:
Because es2015 modules are in strict mode by default, so when we use babel to transpile all our code is strict, without us having to write it everywhere :)